pkg/git/github.go

```diff
@@ -64,13 +65,14 @@ func NewGitHubGITProvider(ctx context.Context, remoteURL, token string) (*GitHub
 }
 
 // CreatePR ...
-func (g *GitHubGITProvider) CreatePR(ctx context.Context, branchName string, auto bool, state *PRState) error {
+//nolint:gocognit //temporary
```

Comment:
Temporary?

Reply:
Yes, other PRs will make changes that will take us back under the limit again.